.github/CONTRIBUTING.md

```diff
@@ -28,6 +28,10 @@ To successfully build the sources on your machine, make sure you've installed th
     - .NET 6
     - .NET 7
 
+Note: .NET has an annual release cycle and we include the preview version in our test matrix.
```

Comment:
if you are here, can you also update the pre-requisites above? we need to have all .NET Frameworks . net452,nrt46,net461,net471 .. we dont list all of them now!

Reply:
net452 is already included! :)

I'm discussing with Raj if we can remove net46 and net461 from this repo.

I can't find any references to net471.

Reply:
![image](https://user-images.githubusercontent.com/5232798/182453649-1b74f01d-ae38-46be-8807-800acb009b7d.png)
These 4 I had to install to build the solution locally. Yes there is no net471

Reply:
Done